hooks/gen_docs.py

```diff
@@ -18,7 +18,7 @@
     "--output",
     str(PATH_CLI_COMMANDS_DOC),
 ]
-subprocess.run(typer_args, shell=True)
+subprocess.run(" ".join(typer_args), shell=True)
```

Comment:
The docs weren't being generated, it's fixed now.

Reply:
this change shouldn't be necessary. `list[str]` is totally fine for `args`

Reply:
ah, it's probably because you're setting `shell=True`. is this necessary?

Reply:
Apparently not :D

fixed.